### PR TITLE
updated tumour grading systems

### PR DESCRIPTION
**Pull request by hknahal**
Plan to include this tumour grading information in documentation section: https://docs.google.com/spreadsheets/d/1uBGaZyyX5_2ZzQCD9Qbv64KG71D72tIYb6Clb1vLSAY/edit?usp=sharing

### CODE REVIEW

**Comment by rosibaj** [comment]
@hknahal should we also add these values as a `codeList`? 
image 

i find without them listed as a codelist, the user has no idea of what possible values to use without looking at the script.

**Comment by hknahal** [comment]
@rosibaj Will they all fit in the box? I agree, the data submitter needs to see the different options. Or we could include a link to a documentation page containing all the tumour grades? But I prefer to squeeze them into this box though. I'll give it a go and see how it looks.

**Comment by rosibaj** [comment]
@hknahal i dont think its any longer than say Primary Site?  it has an expand more/less feature so only 5 will show then they can click to see more

**Comment by rosibaj** [comment]
I mean all values in one codelist

**Comment by hknahal** [comment]
Ok, sounds good. Yeah, I think we can put all the values in one codelist. If the data submitter enters an incorrect grade (for example, they select `Grading system for GNETs` for `tumour_grading_system` which goes from G1-3, but they submit `G4` in the `tumour_grade` field), the submission system will alert them to the error anyways.

File: tests/specimen/tumourGrade.test.js [hunk 1]
```diff
@@ -10,247 +10,146 @@ const name = 'tumour_grade';
...
     }),
   ],
   [
-    'Grading system set to "Nottingham", tumour grade is "G3 (High grade or poorly differentiated)"',
+    'Grading system set to "Nuclear grading for DCIS", tumour grade is "G3"',
     true,
     loadObjects(specimen, {
-      tumour_grading_system: 'nottingham ',
-      tumour_grade: 'G3 (High grade or poorly differentiated)',
+      tumour_grading_system: 'Nuclear grading for DCIS ',
```
Comment (wajiha-oicr) [inline tests/specimen/tumourGrade.test.js:120]: the tests here are missing the word system

Comment (hknahal) [reply tests/specimen/tumourGrade.test.js]: Thanks Wajiha. I fixed it (added `grading` back in)

File: references/validationFunctions/specimen/tumourGrade.js [hunk 2]
```diff
@@ -11,43 +11,91 @@ const validation = ($row, $field) =>
...
+            'g3',
           ];
           break;
-        case 'nottingham':
+        case 'four-tier grading system':
           codeList = [
-            'g1 (low grade or well differentiated)',
-            'g2 (intermediate grade or moderately differentiated)',
-            'g3 (high grade or poorly differentiated)',
+            'gx',
```
Comment (wajiha-oicr) [inline references/validationFunctions/specimen/tumourGrade.js:30]: just a small thing, I see these values used as the same codelist in a few places. maybe you could assign this to a variable and refer to it? 

i.e. before the switch statement you could have `const threeTierGradingList = ['gx','g1','g2','g3']`

then whereever it applies use `codeList = threeTierGradingList`

If you have to slightly the values later, it reduces the likelihood of a mistake

**Pull request merged**